packages/helpers/src/cms/getBackgroundImageUrl.ts

```diff
@@ -0,0 +1,31 @@
+import { CmsBlock, CmsSection } from "@shopware-pwa/types";
+
+const DEFAULT_BG_IMAGE_SIZE = 800;
+function roundUp(num: number) {
+  return num ? Math.ceil(num / 100) * 100 : DEFAULT_BG_IMAGE_SIZE;
+}
+
+const getUrlFromBackgroundImage = (url: string) => {
+  const regex = /(?:\(['"]?)(.*?)(?:['"]?\))/;
```

Comment:
We should change the variable name or add a comment to describe this regex

Reply:
What would be a better variable name?

Reply:
I added tests for the new helper so I guess it is clear what this regex is doing now.